src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -878,13 +887,21 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     this.handleUserSearch(input);
   }
 
-  private onPersonClick(person: IDynamicPerson): void {
+  private onPersonClick(person: IDynamicPerson, e: MouseEvent): void {
+    e.stopPropagation();
```

Comment:
I don't think you need the changes here any more now that you switched to a click handler on input

Reply:
4bc24ba